enterprise/models.py

```diff
@@ -1567,12 +1575,20 @@ def get_course(self, course_key):
 
         Arguments:
             course_key (str): The course key which identifies the course.
+            request (request): Optional argument for detecting if USE_ENTERPRISE_CATALOG flag is active
```

Comment:
Should we remove the docstring comments about the `request` arguments now that it's no longer passed into these methods?